.ci_support/linux_64_.yaml

```diff
@@ -17,7 +17,7 @@ fftw:
 gdal:
 - '3.5'
 geos:
-- 3.11.0
+- 3.11.1
```

Comment:
Error message at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=617459&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=2054 is:

```
gmt: error while loading shared libraries: libgdal.so.31: cannot open shared object file: No such file or directory
```

It might be because the build step is using GDAL 3.5.0, but the runtime is pulling GDAL 3.6.0. Perhaps wait for the GDAL 3.6.0 migration after https://github.com/conda-forge/gdal-feedstock/pull/651?